tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -855,13 +855,7 @@ public void SQLite_InstallAndLogMessageProgrammatically()
 
                 DatabaseTarget testTarget = new DatabaseTarget("TestSqliteTarget");
                 testTarget.ConnectionString = connectionString;
-                string dbProvider = "";
-#if MONO 
-                dbProvider = "Mono.Data.Sqlite.SqliteConnection, Mono.Data.Sqlite";
-#else
-                dbProvider = "System.Data.SQLite.SQLiteConnection, System.Data.SQLite";
-#endif
-                testTarget.DBProvider = dbProvider;
+                testTarget.DBProvider = GetSQLiteDbProvider();
```

Comment:
Moved this code to a separate helper `GetSQLiteDbProvider()` so I could reuse it in my test.

I haven't tested that this works in Mono.